plugin/blocks/flexwall-break/block.json

```
@@ -0,0 +1,8 @@
+{
+  "apiVersion": 2,
+  "title": "FlexWall Break",
+  "name": "flexwall/flexwall-break",
+  "category": "layout",
```

```suggestion
  "category": "widgets",
```
Layout is not a valid category (if you didn't add it). https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#category

---

Or maybe `embed`